### ISSUE TITLE
[1.18] Crash before loading with Tinkers & Hammerlib

### ISSUE THREAD

**Issue by BonaireDreams** [SlimeKnights/TinkersConstruct#4800]
### Minecraft Version

1.18.1

### Forge Version

39.1.2

### Mantle Version

1.8.37

### Tinkers' Construct Version

3.4.2.60

### Describe your issue

Custom modpack Game crashes before loading with Hammerlib 1.18.1-18.1.11 & Tinkers 3.4.2.60
Same modpack configuration works fine with Tinkers 1.18.1-3.4.1.46
Not sure if Tinkers or Hammerlib issue. Opening on their Github as well.

### Crash Report

https://pastebin.com/122v1NTw

### Other mods

Custom modpack. Same modpack works with Tinkers 1.18.1-3.4.1.46

### Tried reproducing with just Tinkers?

No

### Performance Enchancers

None of the above

### Searched for known issues?

Checked pinned issues, Searched open issues, Searched closed issues

**Comment by KnightMiner** [comment]
```
java.lang.NullPointerException: Cannot read field "internal" because the return value of "org.zeith.hammerlib.api.config.ConfigHolder.getCurrent()" is null
```

Stacktrace points to HammerLib. Without more evidence that this is tinkers, going to have to close as a hammerlib bug as I see nothing there that says tinkers is related. Did you try testing without tinkers?

You also are missing the bottom of the crash report.

----

Also, when reporting issues to multiple mods, helpful to link the two issues, as that makes it easier for devs to communicate

**Comment by BonaireDreams** [comment]
Fixed by updating JEI to the last available BETA for 1.18.1 JEI-1.18.1-9.4.1.112. Odd that the pack ran fine for weeks and only crashed when I updated Tinkers to the newer version today. Seems others were getting a similar crash with other mods (Pnuematicraft etc) but it was JEI that needed to be updated to this latest Beta version.
